releasenotes/notes/detect-win-ec2-instance-3e90f6ad7ae03683.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Hostnames starting with ``EC2AMAZ-`` will now have their canonical hostname set as instance id.
```

Comment:
This change is specific to AWS, let's mention that here

Reply:
upgrade comment removed